src/lib/dom.js

```diff
@@ -39,6 +39,13 @@ export function getElement(id) {
 
 export function popup(url, options) {
 
+    // Condition for when dimensions (optional) for pop-up not specified
```

Comment:
I have one small concern here -- if `width === 0` we're just ignoring all of the other options that might be passed too. I think we can probably do this without changing `popup()`

Probably if the code which calls `popup()` just does not pass in `width` and `height` this should work

Right now:

```popup({
  width: x,
  height: y,
  foo: 'bar'
})```

But if we change that to

```popup({
  foo: 'bar'
})```

We should get a full-screen popup